repository-data/application/src/main/resources/hcm-content/taxonomies.yaml

```diff
@@ -0,0 +1,2836 @@
+        /plastic_surgery:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: plastic_surgery
+          locale.en.name: Plastic surgery
+        /psychiatry_of_intellectual_disability__pid_:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: psychiatry_of_intellectual_disability__pid_
+          locale.en.name: Psychiatry of intellectual disability (PID)
+        /public_health__doctor_:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: public_health__doctor_
+          locale.en.name: Public health (doctor)
+        /rehabilitation_medicine:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: rehabilitation_medicine
+          locale.en.name: Rehabilitation medicine
+        /renal_medicine:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: renal_medicine
+          locale.en.name: Renal medicine
+        /respiratory_medicine:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: respiratory_medicine
+          locale.en.name: Respiratory medicine
+        /rheumatology:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: rheumatology
+          locale.en.name: Rheumatology
+        /sport_and_exercise_medicine:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: sport_and_exercise_medicine
+          locale.en.name: Sport and exercise medicine
+        /stroke_medicine:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: stroke_medicine
+          locale.en.name: Stroke medicine
+        /trauma_and_orthopaedic_surgery:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: trauma_and_orthopaedic_surgery
+          locale.en.name: Trauma and orthopaedic surgery
+        /tropical_medicine:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: tropical_medicine
+          locale.en.name: Tropical medicine
+        /urology:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: urology
+          locale.en.name: Urology
+        /vascular_surgery:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: vascular_surgery
+          locale.en.name: Vascular surgery
+      /midwives:
+        jcr:primaryType: hippotaxonomy:category
+        hippotaxonomy:key: midwives
+        locale.en.name: Midwives
+        /midwife:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: midwife
+          locale.en.name: Midwife
+      /nurses:
+        jcr:primaryType: hippotaxonomy:category
+        hippotaxonomy:key: nurses
+        locale.en.name: Nurses
+        /adult_nurse:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: adult_nurse
+          locale.en.name: Adult nurse
+        /children's_nurse:
```

Comment:
Can we remove the trailing `'` from the name and key ?

Reply:
Going through and changing all of these now